pages/apply/start.tsx

```diff
@@ -13,57 +14,65 @@ import {
 import { updateBeforeFirstSave } from '../../lib/store/mainApplicant';
 import { FormID, getFormData } from '../../lib/utils/form-data';
 import processPhonenumber from '../../lib/utils/processPhonenumber';
+import UserErrors from '../../components/errors/user';
 
 const ApplicationStartPage = (): JSX.Element => {
   const router = useRouter();
   const isLoggedIn = useAppSelector((store) => store.cognitoUser?.username);
   const dispatch = useAppDispatch();
   const store = useAppStore();
 
+  const [userError, setUserError] = useState(null);
+
   if (isLoggedIn) {
     router.push('/apply/overview');
   }
 
   const signUp = async (values: FormikValues) => {
     const phone = values.phoneNumber && processPhonenumber(values.phoneNumber);
 
-    await Auth.signUp({
-      username: values.emailAddress,
-      // See https://aws.amazon.com/blogs/mobile/implementing-passwordless-email-authentication-with-amazon-cognito/
-      // on how to generate a random password securely.
-      password: values.password,
-      attributes: {
-        given_name: values.firstName,
-        family_name: values.surname,
-        phone_number: phone, // E.164 number convention
-      },
-    });
-
-    dispatch(
-      updateBeforeFirstSave({
-        person: {
-          title: values.title,
-          firstName: values.firstName,
-          surname: values.surname,
-          dateOfBirth: values.dateOfBirth,
-          gender: values.gender,
-          nationalInsuranceNumber: values.nationalInsuranceNumber
+    try {
+      await Auth.signUp({
+        username: values.emailAddress,
+        // See https://aws.amazon.com/blogs/mobile/implementing-passwordless-email-authentication-with-amazon-cognito/
+        // on how to generate a random password securely.
+        password: values.password,
+        attributes: {
+          given_name: values.firstName,
+          family_name: values.surname,
+          phone_number: phone, // E.164 number convention
         },
-        contactInformation: {
-          emailAddress: values.emailAddress,
-          phoneNumber: phone,
-        },
-      })
-    );
+      });
+
+      dispatch(
+        updateBeforeFirstSave({
+          person: {
+            title: values.title,
+            firstName: values.firstName,
+            surname: values.surname,
+            dateOfBirth: values.dateOfBirth,
+            gender: values.gender,
+            nationalInsuranceNumber: values.nationalInsuranceNumber,
+          },
+          contactInformation: {
+            emailAddress: values.emailAddress,
+            phoneNumber: phone,
+          },
+        })
+      );
 
-    dispatch(createApplication(store.getState().application));
+      dispatch(createApplication(store.getState().application));
 
-    router.push('/apply/verify');
+      router.push('/apply/verify');
+    } catch (error) {
+      setUserError(error.message);
```

Comment:
I think we should declare some generic error messages, rather than relying on anything that comes back via an external service.